### ISSUE TITLE
I can't add a behavior file (sorry, I'm not very good at English).

### ISSUE THREAD

**Issue by astralcrusader** [Goodbird-git/CNPC-EpicFight-Addon#6]
I want to add NPCs from the mod custom NPCs new moves from the mod (I mean the skill books their fashion epic battles), but we'll take an example simpler... let's say I want to remove the backroll (as it was written in the comments to the CNPC-epic fight on the cursed forge). I make a copy of the json behavior file from the datapack, I remove the backroll everywhere, I put the new file back into the archive of the datapack and ... nothing happens - the new behavior model does not appear in the list of NPCs available when configuring (there is standard behavior, the behavior of a skeleton and a samurai, that is, only what originally came with the datapack). Maybe I'm doing something wrong, maybe I'm saving the behavior file wrong, maybe I'm archiving it wrong, maybe I'm installing dependent mods wrong, maybe I'm repenting of my sins wrong... I just don't know anymore, because I'm not very good at all this, and I've searched everything I could, but I haven't found an answer.

![Core mods](https://github.com/user-attachments/assets/2481805b-0f1b-4217-9af8-19be65d35a6f)

**Comment by astralcrusader** [comment]
Ideally, if you are still working on this mod, and if you have the time and opportunity, it would be good to make a section on the wiki about modifying and creating your own datapacks (however, maybe the problem is only with me)

**Comment by Goodbird-git** [comment]
Hello!
Sorry for the very long delay with the response
There is an EpicFight wiki about the datapack editing
https://epicfight-docs.readthedocs.io/Guides/Entities/page1/

**Comment by HDRweidmann** [comment]
Hey, same problem here. Made everything as written in the Wiki. Installed the 4 Mods, and written the Datapack. The Datapack does not show up in the NPC Configuration List, just the usual stuff, Samurai, Skeleton, CustomNPC etc. I havent found any solution. Maybe someone can help. (And yes i wrote it in the new Datapack format and checked, if the code is right)
Tried with different names than "customnpc.json"
![Image](https://github.com/user-attachments/assets/d2c72c79-c47a-4986-af9d-e693e2fc945e)

**Comment by HDRweidmann** [comment]
@astralcrusader Jo, got the solution. Just put it in the datapack folder of your world, not the one in the Modpack Folder. It shows it and now the new configs appear ingame. See picture

![Image](https://github.com/user-attachments/assets/110184eb-d298-4c60-bab0-476962b59fe5)
